[2023-06-29T23:40:57.058+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: execute_python.is_processed manual__2023-06-26T04:46:20.386225+00:00 [queued]>
[2023-06-29T23:40:57.068+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: execute_python.is_processed manual__2023-06-26T04:46:20.386225+00:00 [queued]>
[2023-06-29T23:40:57.068+0200] {taskinstance.py:1308} INFO - Starting attempt 3 of 4
[2023-06-29T23:40:57.096+0200] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): is_processed> on 2023-06-26 04:46:20.386225+00:00
[2023-06-29T23:40:57.101+0200] {standard_task_runner.py:57} INFO - Started process 330310 to run task
[2023-06-29T23:40:57.105+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'execute_python', 'is_processed', 'manual__2023-06-26T04:46:20.386225+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/thesis_container.py', '--cfg-path', '/tmp/tmppohy3a5x']
[2023-06-29T23:40:57.106+0200] {standard_task_runner.py:85} INFO - Job 120: Subtask is_processed
[2023-06-29T23:40:57.148+0200] {task_command.py:410} INFO - Running <TaskInstance: execute_python.is_processed manual__2023-06-26T04:46:20.386225+00:00 [running]> on host DESKTOP-3KJU1M7.
[2023-06-29T23:40:57.243+0200] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='execute_python' AIRFLOW_CTX_TASK_ID='is_processed' AIRFLOW_CTX_EXECUTION_DATE='2023-06-26T04:46:20.386225+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-26T04:46:20.386225+00:00'
[2023-06-29T23:40:57.245+0200] {python.py:183} INFO - Done. Returned value was: False
[2023-06-29T23:40:57.274+0200] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=execute_python, task_id=is_processed, execution_date=20230626T044620, start_date=20230629T214057, end_date=20230629T214057
[2023-06-29T23:40:57.316+0200] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-29T23:40:57.354+0200] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
